poller/poller.go

```diff
@@ -0,0 +1,61 @@
+package poller
+
+import (
+	"time"
+
+	"github.com/godaddy/split-go-serializer/api"
+)
+
+// Poller contains cache data and SplitioAPIBinding
+type Poller struct {
+	Cache              int
+	Error              error
```

Comment:
this should probably be a `chan error`

Reply:
oops miss this message before merging. Not sure why it should be a chan error, but I will definitely revisit this when implementing `pollForChanges()` function.